securedrop/tests/test_journalist_utils.py

```diff
@@ -3,38 +3,8 @@
 import pytest
 import random
 
-from models import RevokedToken
 from sqlalchemy.orm.exc import NoResultFound
 
-from journalist_app.utils import cleanup_expired_revoked_tokens
-
```

Comment:
The test here have been dropped as the table do not exists anymore. The file can probably be deleted.